mlflow/mlflow.go

```diff
@@ -20,6 +20,7 @@ type Client struct {
 	Artifacts   *ArtifactsService
 	Experiments *ExperimentService
 	Metrics     *MetricsService
+	Models      *ModelService
```

Comment:
Since there are "model versions" and "registered models", and the routes are named similarly as well, I think it's best calling this `ModelVersions` and the service `ModelVersionService`, and the file `model_versions.go`.